about.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="icon" href = "assets/Icons/favicon.ico" type = "image/x-icon">
+
+  <link rel="stylesheet" href="./style.css">
+  <!-- COCOGOOSE & LATO FONT -->
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <!-- Link to FontAwesome page -->
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" 
+  integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+  crossorigin="anonymous"/>
+
+</head>
+<body>
+  <header>
+    <nav class="nav-bar">
+      <section class="nav-bar-section1">
+        <div><i onclick='openWindow("erik-fb")' class="fab fa-facebook-f"></i></div>
+        <div><i onclick='openWindow("lore-ig")' class="fab fa-instagram"></i></div>
+        <div>English</div>
+        <div>My Page</div>
+        <div>Logout</div>
+      </section>
+      
+      <section class="nav-bar-section2">
+        <div class="nav-bar-logo">
+         <a href="./index.html"><img src="assets/Logo1.png" alt=""></a> 
+        </div>
+        <ul class="nav-bar-options">
+          <li><a href="./about.html">About</a></li>
```

Comment:
- [ ] As we are already on the about page, it would be intuitive to instead have this link to the index.html page. Then also, make the name be something like "Home". This would really improve the feel and use of your website. Kindly make this small change.

Reply:
**You're right.**

I left a link to the logo as the instructions say but I also added a home button to make it more intuitive.
-
![image](https://user-images.githubusercontent.com/106561762/181612401-027459a3-e6f7-4097-934a-f8a476865ff8.png)